packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -97,8 +108,10 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 
   public render() {
     let rootClass = 'ms-DatePicker';
-    let { firstDayOfWeek, dateRangeType, strings, isMonthPickerVisible, isDayPickerVisible, autoNavigateOnSelection, showGoToToday, highlightCurrentMonth, navigationIcons } = this.props;
-    let { selectedDate, navigatedDate } = this.state;
+    let { firstDayOfWeek, dateRangeType, strings, showMonthPickerAsOverlay, autoNavigateOnSelection, showGoToToday, highlightCurrentMonth, navigationIcons } = this.props;
+    let { selectedDate, navigatedDate, isMonthPickerVisible, isDayPickerVisible } = this.state;
+    let onHeaderClick = showMonthPickerAsOverlay ? this._onHeaderClick : undefined;
+    let showSmallMonthPicker = isMonthPickerVisible && isDayPickerVisible || showMonthPickerAsOverlay && isMonthPickerVisible ? true : false;
```

Comment:
You will want to use parenthesis here to ensure the precedence is correct. Also  no need for ? true : false, it is implied. Also the condition is a bit confusing to me, under which conditions do you want to show the smaller month picker? Should we standardize the size so we don't need to special-case it? In OWA, the day and month picker are both the same size.